README.md

```diff
@@ -10,4 +10,39 @@ See also [pending pull requests](https://github.com/OCFL/extensions/pulls) for e
 
 Community extensions should be written as GitHub flavored markdown in the `docs` directory of this repository. They should be numbered sequentially using a 4-digit, zero-padded prefix; should use hyphens to separate words; and have the `.md` extension.
 
-An example/template is available in this repository as [OCFL Community Extension](docs/0000-example-extension) and is rendered via GitHub pages as https://ocfl.github.io/extensions/0000-example-extension
+An example/template is available in this repository as [OCFL Community Extension](docs/0000-example-extension) and is rendered
+via GitHub pages as https://ocfl.github.io/extensions/0000-example-extension
+
+## Extension Parameters
+
+For efficiency, it is likely that many extension definitions might actually cover a number of variants. Therefore, when an
+extension is referenced, it may be accompanied by a number of parameters that specify the particular variant in use. This
+provides both more effective documention of an OCFL structure but allows the implementation of generic extension code that
+covers a wider variety of use cases. Parameters MUST have be single valued. For each parameter the following properties should
+be defined:    
+
+* Name: A short name for the parameter. Since this has then potential to be used as part of programmatic access the name MUST
+not contain control characters and SHOULD be shorter than 127 characters. The length limit is based on a survey of the defaults for various JSON parsers. 
+* Description: A brief description of the function of the parameter. This should be expanded in the main description of the
+extension which MUST reference all the parameters.
+* Type: Data type for the parameter. In order to allow validation and limit to scope for implementation specific variations,
+parameters are typed.
+  * integer - may be signed or not as specified in the range parameter.
+  * string - aligned with JSON strings, these should be UTF-8 encoded and avoid control characters.  
+  * enumerated - one of an ordered set of labels which MUST conform to the same limitations as parameter names. No specific values are associated with a label other than its ordinality in the set, which is zero-based.
```

Comment:
> No specific values are associated with a label other than its ordinality in the set, which is zero-based.

I am not exactly sure I understand what is trying to be conveyed with this sentence. Could it be clarified?
Are you saying that the labels in the enumeration effectively map to a set of zero-based integers?

Reply:
Yes, that's what i mean but I'm wondering if it is even needed.

Reply:
Dropped.